src/styles/header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+
+  &__logo {
+    position: fixed;
+    top: 10px;
+    width: 40px;
+    margin: 0 auto;
+  }
+}
+
+.nav {
```

Comment:
it's a different bem block, 9'th point in checklist

Reply:
Not fixed